src/instrumentations/xhr.ts

```diff
@@ -709,6 +709,7 @@ export function instrumentXHR(config: XhrConfig) {
           this.addEventListener('readystatechange', () => {
             if (this.readyState === XMLHttpRequest.HEADERS_RECEIVED) {
               const headers = this.getAllResponseHeaders().toLowerCase();
+              currentSpan.setAttribute('headers', headers);
```

Comment:
we want to normalize each header and set it separately (https://github.com/open-telemetry/opentelemetry-js/blob/f3aedb76ff00ca82281f8d2bf94a19714487f1d6/experimental/packages/opentelemetry-instrumentation-http/src/utils.ts#L583)

Reply:
Also any chance we could extract request headers as well ?

Reply:
Due to security and privacy considerations, browsers do not provide an API to access request headers through XMLHttpRequest (XHR). We can only obtain custom headers set by the user, but we currently do not offer a related API to our developers. Therefore, it may not be possible to implement the retrieval of request headers at this time.

Reply:
> we want to normalize each header and set it separately (https://github.com/open-telemetry/opentelemetry-js/blob/f3aedb76ff00ca82281f8d2bf94a19714487f1d6/experimental/packages/opentelemetry-instrumentation-http/src/utils.ts#L583)

I have made the modifications. Could you please review them? Thank you.